src/parser/common/basicParser.ts

```diff
@@ -23,7 +23,8 @@ export default abstract class BasicParser<C = any> {
         parser.removeErrorListeners();
         parser.addErrorListener(new ParserErrorListener(errorListener));
 
-        const parserTree = parser.program();
+        // Note :  needed by pgsql
+        const parserTree = parser.program? parser.program() : parser.root();
```

Comment:
Hi, @salvoravida , In fact, we have modified the antlr4 g4 source file and unified the program as the `root`. you also can change this https://github.com/DTStack/dt-sql-parser/blob/84108968114226023b70819b5451374d42b1d2af/src/grammar/pgsql/PostgreSQLParser.g4#L15,  so that reduce extra code.

Reply:
done